src/bin/ch-remote.rs

```diff
@@ -3,17 +3,23 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+use std::fmt;
+use std::io::Read;
+use std::marker::PhantomData;
+use std::os::unix::net::UnixStream;
+use std::process;
+
 use api_client::simple_api_command;
 use api_client::simple_api_command_with_fds;
 use api_client::simple_api_full_command;
 use api_client::Error as ApiClientError;
 use clap::{Arg, ArgAction, ArgMatches, Command};
 use option_parser::{ByteSized, ByteSizedParseError};
-use std::fmt;
-use std::io::Read;
-use std::marker::PhantomData;
-use std::os::unix::net::UnixStream;
-use std::process;
+use vmm::config::RestoreConfig;
+use vmm::vm_config::{
+    DeviceConfig, DiskConfig, FsConfig, NetConfig, PmemConfig, UserDeviceConfig, VdpaConfig,
+    VsockConfig,
+};
```

Comment:
I suggest we keep at least one prefix here to provide more context, instead of making items inside directly visible :)

```suggestion
use vmm::config as ch_config;
use vmm::vm_config;
```

Reply:
We didn't clearly separate `config.rs` and `vm_comfig.rs`, part of `VmConfig` impl is also in `config.rs`. And we can infer what it is used from the structure name. I don't think it matters.